res/values-tr/strings.xml

```diff
@@ -1091,7 +1091,7 @@ Bu, özel görünümlerde kullanılması *YASAK* olan şeylerin bir listesidir;
      </string>
     <string name="Community">"Topluluk"</string>
     <string name="Buy">"Satın al"</string>
-    <string name="Sell">"Sat"</string>
+    <string name="Sell">"Satış"</string>
```

Comment:
This word has already been translated correctly 
Sat, already is the correct usage in the game